archivers/pom.xml

```diff
@@ -48,7 +48,7 @@
     <parent>
         <groupId>org.jahia.configuration</groupId>
         <artifactId>jahia-configuration-root</artifactId>
-        <version>5.17-SNAPSHOT</version>
+        <version>5.18-SNAPSHOT</version>
```

Comment:
Shouldn't we rather switch to 6.0-SNAPSHOT ?

Reply:
Probably